zuul-simple-webapp/src/main/groovy/filters/route/SimpleHostRoutingFilter.groovy

```diff
@@ -263,14 +263,15 @@ class SimpleHostRoutingFilter extends ZuulFilter {
             if (rebuiltQueryString.length() > 0) {
                 rebuiltQueryString = rebuiltQueryString + "&"
             }
-            def (name,value) = keyPair.split("=", 2)
```

Comment:
Specifically, this was causing an array out of bounds exception if there was no `=` in the query param.